### PR TITLE
feat: pass more information about attachments

### PR DESCRIPTION
**Pull request by VB-at-Bis**
Implements #1021

### CODE REVIEW

File: Infrastructure/arc_lambda/index.js [hunk 1]
```diff
@@ -43,21 +45,82 @@ exports.handler = async(event, context, callback) => {
       output.on('end', function() {
         console.log('Data has been drained');
       });
-      
+      output.on('error', function (err) { 
+        console.log('[OUTPUT STREAM ERROR]', err);
```
Comment (wenzowski) [inline Infrastructure/arc_lambda/index.js:49]: can multiple output events be dispatched? what else can we do with the error other than logging to stderr? should we exit the function?

File: Infrastructure/arc_lambda/index.js [hunk 2]
```diff
@@ -43,21 +45,82 @@ exports.handler = async(event, context, callback) => {
       output.on('end', function() {
         console.log('Data has been drained');
       });
-      
+      output.on('error', function (err) { 
+        console.log('[OUTPUT STREAM ERROR]', err); 
+      });
+
+      archive.on('error', function(err) {
+        console.log('[ARCHIVER ERROR]', err);
```
Comment (wenzowski) [inline Infrastructure/arc_lambda/index.js:53]: can multiple archive events be dispatched? what else can we do with the error other than logging to stderr? should we abort the response with a failure http status code?

File: Infrastructure/arc_lambda/index.js (hunk 1 above)
Comment (VB-at-Bis) [reply Infrastructure/arc_lambda/index.js:49]: This is lambda function, so if we can create archive, we'll add all error logs into that errors.txt; if we fail to create archive then we'll check CloudWatch. This operation (create archive) happens once when intake is closed (invoked by the cron job), so I don't think we need to complicate the function by adding push to SNS to send email etc.

File: Infrastructure/arc_lambda/index.js (hunk 2 above)
Comment (VB-at-Bis) [reply Infrastructure/arc_lambda/index.js:53]: This is lambda function invoked by cron job. Http code would be ignored anyway :) 
I will add documentation that describes steps how we close intake (two cron jobs need to run - one marks applications as received, second generates the archive of attachments)

**Review by marcellmueller — APPROVED** [review]

**Pull request merged**